source/common/upstream/upstream_impl.h

```diff
@@ -577,10 +578,25 @@ class ClusterInfoImpl : public ClusterInfo, protected Logger::Loggable<Logger::I
   TransportSocketMatcher& transportSocketMatcher() const override { return *socket_matcher_; }
   ClusterStats& stats() const override { return stats_; }
   Stats::Scope& statsScope() const override { return *stats_scope_; }
+
+  ClusterRequestResponseSizeOptRef requestResponseSizeStats() const override {
+    if (not optional_cluster_stats_ or not(optional_cluster_stats_->request_response_size_stats_)) {
```

Comment:
although legal C++ we don't typically use this in Envoy style; we would use `!`. Also for the second term, that's a unique_ptr and not a boolean, so it'd be more stylistically consistent to use:
```
optional_cluster_stats_->request_response_size_stats_ != nullptr
```